src/react-apps/applications/runtime/src/shared/components/altinnError.tsx

```diff
@@ -14,6 +14,7 @@ export interface IAltinnErrorProps extends WithStyles<typeof styles> {
   imageUrl?: string;
   imageAlt?: string;
   titleFontWeight?: 'medium';
+  titleColor?: string;
```

Comment:
Not really a fan of extending interfaces of a component just for styles, when we have material UI and can use styling (css).